pkg/syscallfilter/doc.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// The syscalfilter package supports filtering child process events by system call or strace event.
```

Comment:
How will filtering of *child* processes help us? Could we instead filter the *parent* process?

The use case here is someone runs `go test`. Then `go test` compiles and runs the test. The test then starts the syscallfilter command to filter on syscalls of its own tests (the parent process).

Reply:
you should not use the command. You should use the package. The command is mainly to show a way to use the package.